src/components/ProfileUserAvatar/ProfileUserAvatar.js

```diff
@@ -0,0 +1,100 @@
+import getConfig from 'next/config'
+import Image from 'next/image'
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+import { connect } from '~/store'
+import {
+  selectGroupsByUserId,
+  selectUserById,
+  selectDisplayRatByUserId,
+  selectAvatarByUserId,
+  withCurrentUserId,
+  selectCurrentUserHasScope,
+} from '~/store/selectors'
+
+import UploadAvatarModal from '../UploadAvatarModal'
+
+const { publicRuntimeConfig } = getConfig()
+const { appUrl } = publicRuntimeConfig
+
+@connect
+class ProfileHeader extends React.Component {
+  /***************************************************************************\
+    Class Properties
+  \***************************************************************************/
+
+  state = {
+    showUploadAvatar: false
+  }
+
+  /***************************************************************************\
+    Private Methods
+  \***************************************************************************/
+
+  _handleToggleUploadAvatar = () => {
+    this.setState((state) => {
+      return { showUploadAvatar: !state.showUploadAvatar }
+    })
+  }
+
+
+  /***************************************************************************\
+    Public Methods
+  \***************************************************************************/
+
+  render () {
+    const {
+      showUploadAvatar
+    } = this.state
+    const {
+      userAvatar,
+    } = this.props
+
+
+    return (
+      <>
+        <div className="user-avatar" onClick={this._handleToggleUploadAvatar}>
+          <div className="avatar xl">
+            <Image
+              unoptimized
+              alt="User's avatar"
+              height={170}
+              src={`${appUrl}${userAvatar}`}
```

Comment:
relying on `appUrl` here is no longer required. We can feed the avatar path directly into `<Image />`